lib/streamlit/elements/multiselect.py

```diff
@@ -280,6 +292,9 @@ def _multiselect(
         label_visibility: LabelVisibility = "visible",
         ctx: Optional[ScriptRunContext] = None,
         max_selections: Optional[int] = None,
+        placeholder: Optional[str] = "Choose an option",
+        placeholder_no_options: Optional[str] = "No options to select.",
+        placeholder_no_results: Optional[str] = "No results",
```

Comment:
`mypy` seems to be complaining about the types of these parameters. I think these could be made `str`s with default values rather than `Optional[str]`s. Either that or lines 314-316 below can be edited to ensure that the proto fields are always populated with a string type.

(I think that I would slightly prefer the first option).

Reply:
Sorry, forgot to check mypy. The first option was applied.
https://github.com/streamlit/streamlit/pull/6901/commits/8b3fd942e6bea8ae514d0236ac45e365e0a863fe